image.mk

```diff
@@ -4,13 +4,13 @@ include semver.mk
 ifneq (, $(shell which podman 2>/dev/null))
         CONTAINER_TOOL ?= podman
 else
-        CONTAINER_TOOL ?= docker
+        CONTAINER_TOOL ?= docker 
```

Comment:
Additional space can be removed.

Reply:
We can fix this later. We need to get everyone to set their IDEs to trim trailing whitespace.